src/main/java/core/basesyntax/model/User.java

```diff
@@ -53,7 +34,14 @@
                 && Objects.equals(password, user.password)
                 && Objects.equals(age, user.age);
     }
-
+public User(String login , String password , Integer age){
+        this.login = login;
+        this.password = password;
+        this.age = age;
+}
+public User(String login){
+        this.login = login;
+}
```

Comment:
The constructor `User(String login)` initializes only the `login` field. Consider whether default values or additional logic are needed for `password` and `age` to avoid potential `null` values or inconsistent states.